src/main/java/com/hubspot/jinjava/el/ext/JinjavaBeanELResolver.java

```diff
@@ -100,7 +104,15 @@ public Object invoke(
     Class<?>[] paramTypes,
     Object[] params
   ) {
-    if (method == null || RESTRICTED_METHODS.contains(method.toString())) {
+    JinjavaInterpreter interpreter = (JinjavaInterpreter) context
```

Comment:
Do you need to check `null `for the interpreter?

Reply:
I added the null check.